.github/workflows/ci.yaml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  go-tests:
+    name: "Go tests"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v3.5.0
+        with:
+          go-version-file: 'go.mod'
+      - name: Run tests
+        id: runTests
+        run: make test
+        env:
```

Comment:
Looks like one of the tests are being skipped due to missing env vars:

https://github.com/form3tech-oss/go-form3/actions/runs/4185577289/jobs/7252823635#step:4:14

Reply:
Added missing env vars, should be fine now

Reply:
> LGTM other than the comment. Does this close #75?

yes, I'll close it